gcsio/src/test/java/com/google/cloud/hadoop/gcsio/integration/GoogleCloudStorageClientImplIntegrationTest.java

```diff
@@ -343,8 +343,9 @@ public void uploadViaPCUInvalidPartFileNamePrefix() throws IOException {
     byte[] bytesToWrite = new byte[partFileCount * bufferCapacity];
     GoogleCloudStorageTestHelper.fillBytes(bytesToWrite);
     WritableByteChannel writeChannel = gcs.create(resourceId);
-    writeChannel.write(ByteBuffer.wrap(bytesToWrite));
```

Comment:
Why it wasn't showing up in `write` earlier? Is it because actual remote wasn't made till `close` but with pipe removal it will?